assets/js/langRedirector.js

```diff
@@ -0,0 +1,36 @@
+// Wrap all this to not pollute global namespace
+{
+    // Generate an array of languages from page
+    const JDM_LANGS = [...document.querySelectorAll('#language-dropdown > ul > li')].map(el => el.classList[0]);
```

Comment:
Define this as a global constant via `<script>` block in Jekyll instead of parsing languages out of a query selector of language dropdown links.